include/library/spdm_requester_lib.h

```diff
@@ -285,6 +285,8 @@ libspdm_return_t libspdm_get_measurement_ex(void *spdm_context, const uint32_t *
  * @retval RETURN_SECURITY_VIOLATION    Any verification fails.
  **/
 libspdm_return_t libspdm_start_session(void *spdm_context, bool use_psk,
+                                       const void *psk_hint,
```

Comment:
These parameters need to be added to the Doxygen comment above. I can update the API documentation.

Reply:
Added.